SingularityService/src/main/java/com/hubspot/singularity/config/TaskQuickLink.java

```diff
@@ -0,0 +1,35 @@
+package com.hubspot.singularity.config;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * TaskQuickLink defines a "quick-link" button to show on the
+ */
+public class TaskQuickLink {
+  /**
+   * Name of the button label in the SingularityUI interface
+   */
+  @JsonProperty
+  private final String name;
+  /**
+   * Handlebars string to parse to generate a URL for the quick link
+   * (e.g. "http://{{TASK_HOST}}/{{TASK_REQUEST_ID}}")
+   */
+  @JsonProperty
+  private final String url;
+
+  @JsonCreator
+  public TaskQuickLink(@JsonProperty("name") String name, @JsonProperty("url") String url) {
```

Comment:
We talked about this in person, but it'd be nice if we had a way to specify which request type(s) a quick link applies to (i.e. `EnumSet<RequestType> requestTypes`).

The RequestType enum is coming in #348 (not merged yet), so it might make sense to do this in a 2nd PR.